examples/web-wagmi/src/App.tsx

```diff
@@ -68,7 +68,9 @@ const client = createClient({
 const lensConfig: LensConfig = {
   bindings: wagmiBindings(),
   environment: staging,
-  sources: [sources.lenster, sources.orb, 'any-other-app-id'],
+  sources: [],
+  // or narrow to the one you are interested in
+  // sources: [sources.lenster, sources.orb, 'any-other-app-id'],
```

Comment:
can be confusing, especially when you compare the result with `lenster`.
Or when you add a comment and then go to the comments section and publication is not there given we don't add any `appId` in the metadata.